sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -329,14 +461,62 @@ export class DataLakeServiceClient extends StorageClient {
     static fromConnectionString(connectionString: string, options?: StoragePipelineOptions): DataLakeServiceClient;
     generateAccountSasUrl(expiresOn?: Date, permissions?: AccountSASPermissions, resourceTypes?: string, options?: ServiceGenerateAccountSasUrlOptions): string;
     getFileSystemClient(fileSystemName: string): DataLakeFileSystemClient;
+    getProperties(options?: ServiceGetPropertiesOptions): Promise<DataLakeServiceGetPropertiesResponse>;
     getUserDelegationKey(startsOn: Date, expiresOn: Date, options?: ServiceGetUserDelegationKeyOptions): Promise<ServiceGetUserDelegationKeyResponse>;
     listFileSystems(options?: ServiceListFileSystemsOptions): PagedAsyncIterableIterator<FileSystemItem, ServiceListFileSystemsSegmentResponse>;
+    setProperties(properties: DataLakeServiceProperties, options?: ServiceSetPropertiesOptions): Promise<ServiceSetPropertiesResponse>;
     undeleteFileSystem(deletedFileSystemName: string, deleteFileSystemVersion: string, options?: ServiceUndeleteFileSystemOptions): Promise<{
         fileSystemClient: DataLakeFileSystemClient;
         fileSystemUndeleteResponse: FileSystemUndeleteResponse;
     }>;
 }
 
+// @public
+export type DataLakeServiceGetPropertiesResponse = ServiceGetPropertiesHeaders & DataLakeServiceProperties & {
```

Comment:
Better to use type alias to avoid duplicating these interfaces.

Reply:
Another thought, it might be better to keep the definition independent with Blob service, in case storage service may add new interfaces on dfs endpoint to Set/Get service properties in the future.

Reply:
Since Blob has to support HNS enabled accounts, the service properties will always be a superset of that of DFS accounts.
Even if it's not, you can always change the definition of the alias when needed.